.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: python
 env: # These should match the tox env list
+    - TOXENV=py26
     - TOXENV=py27
+    - TOXENV=py33
```

Comment:
I wouldn't bother with py33 or py26 unless you want to put effort into supporting them.  They're both end-of-lifed and anybody sane is going to be using py27, py34, or py35

The coverage issue in py26 is due to a coveragepy bug:

https://bitbucket.org/ned/coveragepy/issues/146/context-managers-confuse-branch-coverage